### DISCUSSION TITLE
Using global stylesheets outside of _app.tsx throws an error

### DISCUSSION THREAD

**Discussion by Norfeldt** [vercel/next.js#25450] [Help]
### What version of Next.js are you using?

10.1.3

### What version of Node.js are you using?

v14.17.0

### What browser are you using?

chrome

### What operating system are you using?

macOS

### How are you deploying your application?

planning on using Vercel

### Describe the Bug

When I try to import a stylesheet that is not a css module in ANY other files it throws an error that it can't be done. This also fits well with the documentation

https://nextjs.org/docs/basic-features/built-in-css-support#adding-a-global-stylesheet

> Due to the global nature of stylesheets, and to avoid conflicts, you may only import them inside pages/_app.js.

BUT I am running some component test that should also have the global stylesheet applied (https://github.com/cypress-io/cypress/discussions/16294) - guess the StoryBook users have the same need. NextJS treats these `*.spec.tsx` files the same.

### Expected Behavior

I should be able to add global css in any component (even thought it's not advised) - showing a warning instead of throwing an error would be more appropriate.

### To Reproduce

I'm using `.scss` BTW

> package.json

```
{
 ...
  "dependencies": {
    "@stripe/react-stripe-js": "^1.4.1",
    "@stripe/stripe-js": "^1.13.2",
    "classnames": "^2.3.1",
    "firebase": "^8.6.1",
    "lodash": "^4.17.21",
    "next": "10.1.3",
    "react": "17.0.1",
    "react-dom": "17.0.1",
    "stripe": "^8.142.0",
    "swr": "^0.5.5"
  },
  "devDependencies": {
    "@cypress/react": "^5.4.2",
    "@cypress/webpack-dev-server": "^1.1.6",
    "@cypress/webpack-preprocessor": "^5.7.0",
    "@testing-library/cypress": "^7.0.6",
    "@types/lodash": "^4.14.168",
    "@types/node": "^14.14.36",
    "@types/react": "^17.0.3",
    "cypress": "7.3.0",
    "cypress-localstorage-commands": "^1.4.4",
    "cypress-plugin-stripe-elements": "^1.0.1",
    "html-webpack-plugin": "4",
    "sass": "^1.32.8",
    "styled-components": "^5.2.3", // 👈 using this as a way to get around the limitation
    "styled-jsx": "^3.4.4",
    "typescript": "^4.2.3",
    "webpack": "4",
    "webpack-dev-server": "3"
  }
}
```

> tsconfig.json

```
{
  "compilerOptions": {
    "target": "es5",
    "downlevelIteration": true,
    "lib": ["dom", "dom.iterable", "esnext"],
    "allowJs": true,
    "skipLibCheck": true,
    "strict": false,
    "forceConsistentCasingInFileNames": true,
    "noEmit": true,
    "esModuleInterop": true,
    "module": "esnext",
    "moduleResolution": "node",
    "resolveJsonModule": true,
    "isolatedModules": true,
    "jsx": "preserve",
    "baseUrl": ".",
    "paths": {
      "~/*": ["*"]
    },
    "types": [
      "cypress",
      "@testing-library/cypress",
      "cypress-localstorage-commands",
      "cypress-plugin-stripe-elements"
    ]
  },
  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx"],
  "exclude": ["node_modules"]
}
```

**Comment by Norfeldt** [comment]
@timneutkens Why is a bug like this considered a discussion?

**Reply by fabb** [reply]
when it's specified like this in the documentation, it's not a bug, it's a feature. i guess it was moved to discussions because you are requesting a feature change which has to be discussed first.
